Blis/src/BlisModel.cpp

```diff
@@ -1787,8 +1787,8 @@ BlisModel::feasibleSolution(int & numIntegerInfs, int & numObjectInfs)
 	std::cout << "FEASIBLE SOL: numUnsatisfied = " 
 		  << numUnsatisfied << std::endl;
     }
-
-    if (!numUnsatisfied) {
+    //sahar
+    //if (!numUnsatisfied) {
         sol = userFeasibleSolution(getLpSolution(), userFeasible);
```

Comment:
I commented out this condition to always step into the user-defined feasibility function. In the case of MIPs, this does not affect the solution of the problem, but it may increase the solution time a little. One thing that I am not sure about it is setting the value of `numBranchResolve_` [here ](https://github.com/coin-or/CHiPPS-BLIS/pull/8/commits/b714e6af651704d0b019ad4495cbbe7529fbdb17#diff-e99049c7749d668bf0984f33bbc3ec55R1798). Without commenting out [this](https://github.com/coin-or/CHiPPS-BLIS/pull/8/commits/b714e6af651704d0b019ad4495cbbe7529fbdb17#diff-e99049c7749d668bf0984f33bbc3ec55R1791), the value of `numBranchResolve_` is 0 when `numUnsatisfied` is non-zero, but by commenting out that condition, the value of this variable is set to 10. I cannot remember the details, but as I remember, it should not affect the solution of problem.